src/SixLabors.Fonts/AppliedFontStyle.cs

```diff
@@ -17,6 +17,8 @@ internal struct AppliedFontStyle
         public int End;
         public bool ApplyKerning;
 
+        // TODO: This should accept all the glyphs in the range to
+        // allow complex substitutions.
         public GlyphMetrics[] GetGlyphLayers(CodePoint codePoint, ColorFontSupport colorFontOptions)
```

Comment:
@tocsoft By my understanding this is our injection point for identifying the correct glyphs for each codepoint. 

It looks like we need to load the entire codepoint buffer though before laying things out in order to remap the glyph ids in context (_we could potentially remove codepoints_).  

I'm seeing though in the run that we load up a style for the entire buffer and never reach the end of that style even though we check per codepoint below. Am I reading that correctly?